app/services/idv/steps/inherited_proofing/verify_info_step.rb

```diff
@@ -5,11 +5,11 @@ class VerifyInfoStep < InheritedProofingBaseStep
         STEP_INDICATOR_STEP = :verify_info
 
         def self.analytics_visited_event
-          :idv_doc_auth_verify_visited
+          :idv_inherited_proofing_verify_visited
         end
```

Comment:
is there a reason this was changed away from the equivalent step in DocAuth and IPP? I think that we could share these and share the documentation

Reply:
> @zachmargolis are you saying this is unnecessary by virtue of the `analytics_id` that is automatically added to step events within the flow state machine?

Yup! That's what I'm saying. When it's possible to share steps/events across flows, I think we should, because then we can compare them easily ("VAIP passes verify step at 50% vs Docauth passing at 40%" or something like that)

Reply:
Restored to original doc auth events to share analytics